tools/pyFlashGui/pyFlashGui.py

```diff
@@ -108,7 +108,7 @@ def init(self):
                 # If the serial port exists but isn't in swadgeSerialPorts
                 if serialPort not in self.swadgeSerialPorts:
                     # If the VID and PID match the ESP32-S2's values, use it
-                    if(serialPort.vid == 0x303A and serialPort.pid == 0x0002):
+                    if(serialPort.vid == 0x1209 and serialPort.pid == 0x4269):
```

Comment:
Uh, it may have been a mistake to change this one. It may still use Espressif's VID/PID in programming mode